customerrors/errors.go

```diff
@@ -4,8 +4,8 @@ import "errors"
 
 var (
 	ErrInvalidEmailPassword = errors.New("invalid email or password")
-
-	NoUsersExist = errors.New("No users exist at present")
-
-	NoAssetsExist = errors.New("No assets exist")
+	NoUsersExist            = errors.New("No users exist at present")
```

Comment:
explain the purpose of two diff errors NoUsersExist & UserDoesNotExist

Reply:
I had made this from different branches. Will change code to keep only one now.